core/domain/question_validators.py

```diff
@@ -0,0 +1,281 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Validators for question models."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.domain import base_model_validators
+from core.domain import question_domain
+from core.domain import question_fetchers
+from core.domain import question_services
+from core.domain import skill_fetchers
+from core.platform import models
+
```

Comment:
There shouldn't be a newline here, right? (as the above and below imports are local imports)

I was expecting pre-push lint test to catch this.

Reply:
Done. Sorry, I'm a bit rusty on the import format rules, but I'll catch up quick.